src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -1064,7 +1248,7 @@ TxQ::accept(Application& app,
                 ledgerChanged = true;
             }
             else if (isTefFailure(txnResult) || isTemMalformed(txnResult) ||
-                isTelLocal(txnResult) || candidateIter->retriesRemaining <= 0)
+                candidateIter->retriesRemaining <= 0)
```

Comment:
I realized that throwing a candidate away on an `isTelLocal` result could result in perfectly good transactions getting dropped if the local (or network) server load shoots up, through no fault of the user. Removing this check will hold on to the tx, giving load a chance to come back down. If load doesn't go down, it'll get dropped soon enough, and remaining txs for this account will get dropped even faster.